example/centos/centos-9-aarch64-tar.yaml

```diff
@@ -1,6 +1,7 @@
 otk.version: "1"
 
 otk.define:
+  architecture: aarch64
```

Comment:
AFAICT the tar image is now fully identical:
```
$ diff -u centos-9-aarch64-tar.yaml centos-9-x86_64-tar.yaml 
--- centos-9-aarch64-tar.yaml	2024-10-01 16:18:33.052687010 +0200
+++ centos-9-x86_64-tar.yaml	2024-10-01 16:18:33.052687010 +0200
@@ -1,7 +1,7 @@
 otk.version: "1"
 
 otk.define:
-  architecture: aarch64
+  architecture: x86_64
   modifications:
     locale: C.UTF-8
     timezone: America/New_York
```
I assume there will be a followup that extracts tar into a common file or something?

Reply:
Yep!